src/autocomplete.js

```diff
@@ -30,16 +42,34 @@ var Autocomplete = function() {
 };
 
 (function() {
+    this.$onPopupChange = function(hide) {
+        if (hide) {
+            if (this.inlineEnabled) {
+                this.inlineRenderer.show(this.editor, null, prefix);    
+            }
+            this.hideDocTooltip();
+        } else {
+            this.tooltipTimer.call(null, null);
+            if (this.inlineEnabled) {
+                var completion = hide ? null : this.popup.getData(this.popup.getRow());
```

Comment:
I think `hide` never can be true here, maybe worth to split into two functions to make it easier to follow?

Reply:
Sure, I'll split it